common/app/views/fragments/atoms/atom.scala.html

```diff
@@ -1,15 +1,12 @@
 @import model.content.MediaWrapper
-@import model.content.QandaAtom
-@import model.content.TimelineAtom
-@import model.content.GuideAtom
-@import model.content.ProfileAtom
 @(model: _root_.model.content.Atom, shouldFence: Boolean, amp: Boolean, mediaWrapper: Option[MediaWrapper])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
 @import _root_.model.ShareLinkMeta
-@import _root_.model.content.{InteractiveAtom, MediaAtom, Quiz, StoryQuiz, StoryQuestionsAtom}
+@import _root_.model.content.{InteractiveAtom, MediaAtom, Quiz, StoryQuiz, StoryQuestionsAtom, TimelineAtom, GuideAtom, ProfileAtom, QandaAtom}
 
 @{
     model match {
-        case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, showResults = false, ShareLinkMeta(Nil, Nil))
+        case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, showResults = false, quiz.shareLinks)
```

Comment:
should we explicitly name the `shareLinks` field since this is what we do for `maybeResults` and `showResults`